packages/dashboard-core-plugins/src/linker/LinkerOverlayContent.tsx

```diff
@@ -139,11 +168,68 @@ export class LinkerOverlayContent extends Component<
     this.setState({ mode: 'select' });
   }
 
+  handleResize(): void {
+    const { dialogX, dialogY } = this.state;
+    const boundedDialogX = Math.min(
+      Math.max(0, dialogX ?? 0),
+      window.innerWidth - (this.dialogRef.current?.offsetWidth ?? 0)
```

Comment:
If you drag the dialog to the very bottom right, then resize, the dialog becomes all bunched up. May need some more constraints in CSS to ensure it maintains it's width/height.